services/document-manager/backend/app/docman/utils/document_upload_helper.py

```diff
@@ -0,0 +1,148 @@
+import base64
+import os
+import time
+from urllib.parse import urlparse
+from wsgiref.handlers import format_date_time
+
+import requests
+from app.config import Config
+from app.constants import OBJECT_STORE_PATH, OBJECT_STORE_UPLOAD_RESOURCE, FILE_UPLOAD_SIZE, FILE_UPLOAD_OFFSET, \
+    FILE_UPLOAD_PATH, FILE_UPLOAD_EXPIRY, TIMEOUT_24_HOURS, TUS_API_VERSION, TUS_API_SUPPORTED_VERSIONS, \
+    FORBIDDEN_FILETYPES
+from app.extensions import cache
+from flask import request, current_app, make_response, jsonify
+from werkzeug.exceptions import BadRequest, NotFound, RequestEntityTooLarge, InternalServerError, BadGateway
+
+CACHE_TIMEOUT = TIMEOUT_24_HOURS
+
+
+class DocumentUploadHelper:
```

Comment:
Nice!  I was confused at first by the deletion of all the code from document.py.